cnd/src/facade.rs

```diff
@@ -32,15 +34,24 @@ impl Facade {
             .await
     }
 
-    /// Returns the Bitcoin median time past, used for nLockTime and
-    /// CheckLockTimeVerify.
-    pub async fn bitcoin_median_time_past(&self) -> anyhow::Result<Timestamp> {
-        self.connectors.bitcoin.median_time_past().await
+    /// Returns the current Bitcoin median time past.
+    pub async fn bitcoin_median_time(&self) -> anyhow::Result<Timestamp> {
```

Comment:
@D4nte and I back and forth'ed on the '_past' part of median time past. I pushed for its inclusion because the term is from the bitcoin ecosystem, they refer to it as 'median time past' not as 'median time'.

Reply:
Really? That is such a confusion term :D

Didn't know, will change back :)